### DISCUSSION TITLE
Duvida CSS

### DISCUSSION THREAD

**Discussion by VNeris** [VNeris/blue-lock-blog#3] [Q&A] [answered]
Como faço para criar um background com gradiente?

**Comment by jneris-dev** [comment, marked as answer]
Existem diversas ferramentas para gerar um background gradiente. Mas basicamente o codigo ficara assim:

`background: linear-gradient(90deg, rgba(2,0,36,1) 0%, rgba(9,9,121,1) 35%, rgba(0,212,255,1) 100%);`

https://cssgradient.io/
